portal-kernel/src/com/liferay/asset/kernel/model/BaseAssetRenderer.java

```diff
@@ -219,6 +219,15 @@ public String getURLImagePreview(PortletRequest portletRequest)
 		return getThumbnailPath(portletRequest);
 	}
 
+	@Override
+	public String getURLShare(
+			LiferayPortletRequest liferayPortletRequest,
+			LiferayPortletResponse liferayPortletResponse)
+		throws Exception {
+
+		return getURLView(liferayPortletResponse, WindowState.MAXIMIZED);
```

Comment:
Hi @robertoDiaz, change looks good to me. I'm just unsure about returning the urlView by default.

@ealonso, what do you think?

Reply:
if you prefer, as talked in slack, we could return a null (or an empty string), as you prefer

Thanks!

Reply:
Do we want to support it for all Assets?

Reply:
Hi @ealonso 

It is likely we'll want to do that in the future.

Looks similar to other assetRenderer methods  use cases as com.liferay.asset.kernel.model.BaseAssetRenderer#getURLDownload (that, by now, is only implemented in DLFileEntryAssetRenderer)

In this case we also like to use the asset renderder to have a consistent API in SharingJavaScriptFactory (passing the className and classPK instead of the URL itself).

Thanks!

Reply:
Got it!

In that casi I agree that it's better return the null 👍 

I'll send the changes returning the null. Are you ok with that??

Reply:
Yes

Reply:
I'll resolve the conversation once resend

Thank you so much @ealonso! 😄